RoboFile.php

```diff
@@ -557,18 +557,42 @@ public function releaseCheckIssues($version = null) {
   }
 
   public function releasePrepareGit() {
+    // make sure working directory is clean
     $git_status = $this->taskGitStack()
       ->printOutput(false)
       ->exec('git status --porcelain')
       ->run();
     if (strlen(trim($git_status->getMessage())) > 0) {
       throw new \Exception('Please make sure your working directory is clean before running release.');
     }
+    // checkout master and pull from remote
     $this->taskGitStack()
       ->stopOnFail()
       ->checkout('master')
       ->pull()
       ->run();
+    // make sure release branch doesn't exist on github
+    $release_branch_status = $this->taskGitStack()
+      ->printOutput(false)
+      ->exec('git ls-remote --heads git@github.com:mailpoet/mailpoet.git release')
+      ->run();
+    if (strlen(trim($release_branch_status->getMessage())) > 0) {
+      throw new \Exception('Delete old release branch before running release.');
+    }
+    $git_status = $this->taskGitStack()
```

Comment:
This checks local "release" branch and deletes it if it exists? Comment would be useful, Git's CLI commands are incredibly unintuitive.